test/mpi/maint/test_coll_algos.sh

```diff
@@ -28,18 +27,20 @@ for algo_name in ${algo_names}; do
                 env="${testing_env} env=MPIR_CVAR_IBCAST_INTRA_ALGORITHM=${algo_name} "
                 env+="env=MPIR_CVAR_IBCAST_TREE_TYPE=${tree_type} "
                 env+="env=MPIR_CVAR_IBCAST_TREE_KVAL=${kval} env=MPIR_CVAR_IBCAST_TREE_PIPELINE_CHUNK_SIZE=4096 "
```

Comment:
I don't think `+=` is portable; use `env="$env ..."`

Reply:
Updated this.